support-frontend/assets/helpers/redux/checkout/addressMeta/state.ts

```diff
@@ -15,23 +17,3 @@ export const initialState: AddressMetaState = {
 	billingAddressMatchesDelivery: true,
 	deliveryAgent: { isLoading: false, response: undefined },
 };
-
```

Comment:
I moved some code related to fetching the delivery agents from the server into a separate file so it could be used by both checkouts without all the other Redux related stuff